frontend/dist/index.html

```diff
@@ -83,31 +83,25 @@
   <div id='content'></div>
   <div id='welcome'><h1>Welcome to Tracim dev</h1></div>
 
-  <!--
-  INFO - CH - 2021-01-14
-  since servdev uses standalone frontend_lib build, you should not include frontend_lib directly in the page
-  it is already in the bundles
-  same for frontend_vendor
-  -->
-<!--  <script type='text/javascript' src='/app/tracim_frontend_vendors.js'></script>-->
-<!--  <script type='text/javascript' src='/app/tracim_frontend_lib.lib.js'></script>-->
-
-  <script type='text/javascript' src='/app/tracim_frontend_lib.style.js'></script>
-
-  <script type='text/javascript' src='/app/file.app.standalone.js'></script>
-  <script type='text/javascript' src='/app/html-document.app.standalone.js'></script>
-  <script type='text/javascript' src='/app/thread.app.standalone.js'></script>
-  <script type='text/javascript' src='/app/collaborative_document_edition.app.standalone.js'></script>
-  <script type='text/javascript' src='/app/admin_workspace_user.app.standalone.js'></script>
-  <script type='text/javascript' src='/app/agenda.app.standalone.js'></script>
-  <script type='text/javascript' src='/app/folder.app.standalone.js'></script>
-  <script type='text/javascript' src='/app/share_folder.app.standalone.js'></script>
-  <script type='text/javascript' src='/app/workspace.app.standalone.js'></script>
-  <script type='text/javascript' src='/app/workspace_advanced.app.standalone.js'></script>
-  <script type='text/javascript' src='/app/gallery.app.standalone.js'></script>
-
-  <script type='text/javascript' src='/assets/tracim.vendors~app.js'></script>
-  <script type='text/javascript' src='/assets/tracim.app.js'></script>
+  <script type='text/javascript' src='/app/tracim_frontend_vendors.js'></script>
+  <script type='text/javascript' src='/app/tracim_frontend_lib.optimized.lib.js'></script>
+
+  <script type='text/javascript' src='/app/tracim_frontend_lib.optimized.style.js'></script>
+
+  <script type='text/javascript' src='/app/file.app.optimized.js'></script>
+<!--  <script type='text/javascript' src='/app/html-document.app.standalone.js'></script>-->
+<!--  <script type='text/javascript' src='/app/thread.app.standalone.js'></script>-->
+<!--  <script type='text/javascript' src='/app/collaborative_document_edition.app.standalone.js'></script>-->
+<!--  <script type='text/javascript' src='/app/admin_workspace_user.app.standalone.js'></script>-->
+<!--  <script type='text/javascript' src='/app/agenda.app.standalone.js'></script>-->
+<!--  <script type='text/javascript' src='/app/folder.app.standalone.js'></script>-->
+<!--  <script type='text/javascript' src='/app/share_folder.app.standalone.js'></script>-->
+<!--  <script type='text/javascript' src='/app/workspace.app.standalone.js'></script>-->
+<!--  <script type='text/javascript' src='/app/workspace_advanced.app.standalone.js'></script>-->
+<!--  <script type='text/javascript' src='/app/gallery.app.standalone.js'></script>-->
```

Comment:
Are those lines leftovers?

Reply:
mmh yes, I will update them with the update of each apps

Reply:
fixed